.clang-format

```diff
@@ -94,9 +94,7 @@ ColumnLimit:     120
 # DisableFormat:   false
 # ExperimentalAutoDetectBinPacking: false
 
-# TRANSITION, LLVM-39247, fixed in clang-format 10.
-# FixNamespaceComments: true
-FixNamespaceComments: false
+FixNamespaceComments: true

```

Comment:
This should actually be commented out. The non-obvious convention in this file is that the default values all appear as comments, and then customized values appear below them. Because the options above and below are also defaults, there should be no empty lines around this.